### ISSUE TITLE
[Provider Registration] Provider Creation limits 

### ISSUE THREAD

**Issue by wilwade** [LibertyDSNP/frequency#321]
Frequency needs an easy way to figure out who is a provider and what the gates are to become one. 
In the short term, we want to make this a governance action, but still be frictionless.

Related: #578

## User Journey (Spike: `spike/governance-based-registrations`)
- User creates a MSA
- User wants to be a provider
- User uses an MSA control key to sign the `create_provider_via_governance` payload
- User asks off-chain (discord, email, etc...) one Frequency Council member executes the extrinsic on their behalf.
- The end.

## User Journey (Spike + Small changes)
- User creates a MSA
- User wants to be a provider
- User asks off-chain (discord, email, etc...) one Frequency Council member votes to create a motion to grant them provider status
- The Frequency Council member creates and votes for for the proposal
- Anyone (likely the one voter) closes the ^ proposal
- Proposal executes and creates provider for original user's MSA
- ALT: User does full referendum process.

## User Journey (User initiated Option) (@rlaferla to look at time boxed to 1-2 days)
- User creates a MSA
- User wants to be a provider
- *User calls extrinsic* that generates a proposal in the Frequency Council
- At least One Frequency Council member votes for the proposal
- Anyone (likely the one voter) closes the ^ proposal
- Proposal executes and creates provider for original user's MSA
- ALT: User does full referendum process.

## Long term: User Journey
- User creates a MSA
- User wants to be a provider
- User stakes/pays fee/etc... and executes extrinsic and becomes provider. The end.

## Acceptance Criteria

- Only a frequency council member can approve an MSA to become a provider
- Local and rococo also allow anyone to become an provider
- Any associated with the MSA must have requested provider creation 
- Documentation updated to reflect

**Comment by enddynayn** [comment]
Is this fee a bond or a one time fee?

**Comment by wilwade** [comment]
@enddynayn Good catch. Added AC for burning it.

**Comment by enddynayn** [comment]
I think we should consider also moving this into its own story:
> The fee should be deducted from the sender's account and burned so that after calling this extrinsic, the account balance should be less those fees.

**Comment by wilwade** [comment]
> I think we should consider also moving this into its own story:
> 
> > The fee should be deducted from the sender's account and burned so that after calling this extrinsic, the account balance should be less those fees.

@enddynayn I think this is a critical part. I suppose the fee config setup could be separate, but that would be moving ` The fee should be settable by config/governance (see MaxMessagePayloadSizeBytes)` to a separate story. That said I really don't like having a configuration item that isn't used. That could be confusing if something happened and the stories got split in time.

**Comment by rlaferla** [comment]
Blocked by https://github.com/LibertyDSNP/frequency/issues/965

**Comment by rlaferla** [comment]
We now have something basic working but it needs refinement.

**Comment by rlaferla** [comment]
Blocked by https://github.com/LibertyDSNP/frequency/issues/1077
